IdentityCore/src/cache/accessor/MSIDDefaultTokenCacheAccessor.m

```diff
@@ -110,28 +117,10 @@ - (BOOL)saveSSOStateWithConfiguration:(MSIDConfiguration *)configuration
 
     MSID_LOG_VERBOSE(context, @"(Legacy accessor) Saving SSO state");
 
-    BOOL result = [self saveIDTokenWithConfiguration:configuration response:response context:context error:error];
-    result &= [self saveRefreshTokenWithConfiguration:configuration response:response context:context error:error];
+    BOOL result = [self saveRefreshTokenWithConfiguration:configuration response:response context:context error:error];
     result &= [self saveAccountWithConfiguration:configuration response:response context:context error:error];
 
-    if (!result)
-    {
-        return NO;
-    }
-
-    for (id<MSIDCacheAccessor> accessor in _otherAccessors)
-    {
-        if (![accessor saveSSOStateWithConfiguration:configuration
-                                            response:response
-                                             context:context
-                                               error:error])
-        {
-            MSID_LOG_WARN(context, @"Failed to save SSO state in other accessor: %@", accessor.class);
-            MSID_LOG_WARN_PII(context, @"Failed to save SSO state in other accessor: %@, error %@", accessor.class, *error);
-        }
-    }
-
-    return YES;
+    return result;
```

Comment:
Do we no longer want to store SSO on other accessors?

Reply:
Discussed offline, but also adding here:
* MSAL writing cache items in legacy ADAL format was redundant, because ADAL unified can read cache from MSAL format as well. We can always add it back in future if we find a need for scenario, but didn't want to add it unless we find a scenario. 
* Basically, ADAL now writes cache in both legacy ADAL and new default format, but MSAL only in default format. They still both can read cache items from both formats.